packages/why3find/why3find.1.0/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Why3 Package Manager"
+description:
+  "The why3find utility is designed for managing packages for why3 developpers and associated OCaml extracted code."
+maintainer: ["benjamin.jorge@cea.fr" "loic.correnson@cea.fr"]
+authors: [
+  "Loïc Correnson <loic.correnson@cea.fr>"
+  "Benjamin Jorge <benjamin.jorge@cea.fr>"
+]
+license: "LGPL-2.1-only"
+tags: ["topics" "why3"]
+homepage: "https://git.frama-c.com/pub/why3find"
+doc: "https://git.frama-c.com/pub/why3find"
+bug-reports: "https://git.frama-c.com/pub/why3find/issues"
+depends: [
+  "dune" {>= "3.1" & >= "3.5"}
```

Comment:
```suggestion
  "dune" {>= "3.5"}
```
To fix this you need to remove `dune` from the dependencies in the `dune-project` file. It will be added to the dependencies list automatically with the correct bound

Reply:
Is it important that I make this change in the source tarball or is it okay to just modify the constraint here ?

Reply:
Ok to modify it here, but please fix it also on the dev branch of your repository